### ISSUE TITLE
Float aspect ratios wont work.

### ISSUE THREAD

**Issue by lucianomunoz** [cgwire/kitsu#940]
**Context**

Studio name:  Pataz
Kitsu version: 0.15.4
Production type:  Short film

**Describe the bug**
When first creating the project you can input aspect ratios but it wont let you input aspect ratios that aren't integer: like 4:3 or 16:9 the most classical for tv, but in film aspect ratios often used are 2.39 : 1 (the one we're using), here are a list of some of the most common ones: 

![image](https://user-images.githubusercontent.com/5134226/188027435-30445f5e-5422-4daf-8a40-a0b3bb4bfc74.png)

Basically the problem is that inputting a float value just wont let you complete the form and continue.

**Comment by frankrousseau** [comment]
While waiting for the fix, you can write another ratio by editing the production in the production list page.

**Comment by frankrousseau** [comment]
It will be fixed in next release.

**Comment by frankrousseau** [comment]
It should be fixed. Can you try again?
